wgkex/worker/netlink_test.py

```diff
@@ -1,24 +1,21 @@
 """Unit tests for netlink.py"""
 
-import unittest
-import mock
-from datetime import timedelta
-from datetime import datetime
-
 # pyroute2 decides imports based on platform. WireGuard is specific to Linux only. Mock pyroute2.WireGuard so that
 # any testing platform can execute tests.
 import sys
+import unittest
+from datetime import datetime, timedelta
 
+import mock
 import pyroute2.netlink.exceptions as pyroute2_netlink_exceptions
 
 pyroute2_module_mock = mock.MagicMock()
 pyroute2_module_mock.netlink.exceptions = pyroute2_netlink_exceptions
 sys.modules["pyroute2"] = pyroute2_module_mock
 sys.modules["pyroute2.netlink"] = mock.MagicMock()
-from pyroute2 import WireGuard
-from pyroute2 import IPRoute
+from pyroute2 import IPRoute, WireGuard  # noqa: E402
 
-from wgkex.worker import netlink
+from wgkex.worker import netlink  # noqa: E402
```

Comment:
I had to silence these two issues because actually the imports *cannot* be re-sorted – they need to be below the mocking code.